ydb/library/yql/providers/pq/async_io/dq_pq_read_actor.cpp

```diff
@@ -374,26 +376,30 @@ class TDqPqReadActor : public NActors::TActor<TDqPqReadActor>, public IDqCompute
             TInstant::Now());
     }
 
-    NYdb::NPersQueue::TReadSessionSettings GetReadSessionSettings() const {
-        NYdb::NPersQueue::TTopicReadSettings topicReadSettings;
+    NYdb::NTopic::TReadSessionSettings GetReadSessionSettings() const {
+        NYdb::NTopic::TTopicReadSettings topicReadSettings;
         topicReadSettings.Path(SourceParams.GetTopicPath());
         auto partitionsToRead = GetPartitionsToRead();
         SRC_LOG_D("PartitionsToRead: " << JoinSeq(", ", partitionsToRead));
         for (const auto partitionId : partitionsToRead) {
-            topicReadSettings.AppendPartitionGroupIds(partitionId);
+            topicReadSettings.AppendPartitionIds(partitionId);
         }
 
-        return NYdb::NPersQueue::TReadSessionSettings()
-            .DisableClusterDiscovery(SourceParams.GetClusterType() == NPq::NProto::DataStreams)
+        Y_UNUSED(RangesMode);
```

Comment:
So it's better to remove this class member. It was a temporary crutch as far as I remember